opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -995,8 +995,15 @@ def record_exception(
                 type(exception), value=exception, tb=exception.__traceback__
             )
         )
+        module = type(exception).__module__
+        qualname = type(exception).__qualname__
+        exception_type = (
+            f"{module}.{qualname}"
+            if module and module != "builtins"
```

Comment:
Why not use `builtins` as part of the qualified name?

Reply:
The example on [specs](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/exception.md) uses `OSError` as an example and I think it'll be more natural for Python developers.

Reply:
Mainly because of readability and this [statement](https://peps.python.org/pep-0737/#add-pytype-getfullyqualifiedname-function)